### PR TITLE
Visually refactor the extension

### PR DESCRIPTION
**Pull request by frm**
Why:

* The codebase was messy with a lot of random styles and cluttered code.

This change addresses the need by:

* Adding `styled-components`.
* Refactoring most of the shared code into separate components.
* Applying consistent styles, sizes and spacings across components.

### CODE REVIEW

**Review by boymaas — APPROVED** [review]

File: data_host/chrome_extension/src/pages/Popup/components/Form/index.tsx [hunk 1]
```diff
@@ -3,6 +3,10 @@ import { useDispatch } from 'react-redux';
 import { Form as FinalForm, Field, FormRenderProps } from 'react-final-form';
 import { FormApi } from 'final-form';
 
+import Error from '../../../../components/Error';
```
Comment (shelbyd) [inline data_host/chrome_extension/src/pages/Popup/components/Form/index.tsx:6]: I'm generally not a fan of importing from relative paths for parent directories. If we move this file and not the other one, this import needs to change. it's also really unclear where this is coming from. 

Are we able to import from src/components/Spacing instead? If not, can we change some config so we can?

**Review by shelbyd — CHANGES_REQUESTED** [review]
nit: I believe the term "Refactor" should only be used when only changing the internal structure of code and not the user-facing behavior. Just know if I ever call something refactoring (and I will a lot) I mean explicitly to not change anything out the external facing behavior.

Perhaps this could be considered a visual refactoring insofar as changing the visual design without changing functionality. Consider changing the title, nbd if you want to keep as is.

File: data_host/chrome_extension/src/pages/Popup/components/Form/index.tsx (hunk 1 above)
Comment (frm) [reply data_host/chrome_extension/src/pages/Popup/components/Form/index.tsx]: Just pushed configuration including path aliases 👍

**Comment by frm** [comment]
Closing due to changing focus to [Fractal Wallet](https://github.com/trustfractal/wallet)

**Pull request closed without merging**